src/sentry/static/sentry/app/views/groupDetails/shared/groupDetails.jsx

```diff
@@ -199,15 +201,23 @@ const GroupDetails = createReactClass({
     } else if (this.state.loading || !group) return <LoadingIndicator />;
 
     return (
-      <DocumentTitle title={this.getTitle()}>
-        <div className={this.props.className}>
-          <GroupHeader params={params} project={project} group={group} />
-          {React.cloneElement(this.props.children, {
-            group,
-            project,
-          })}
-        </div>
-      </DocumentTitle>
+      <React.Fragment>
+        <GlobalSelectionHeader
```

Comment:
Hmm this is inside of the shared group details, won't this show on old sentry pages?

Reply:
egh yes. thank you. the issue is getting the group and it's project but I can just test for sentry 10 again here.